_posts/2021-04-13-upgrade-notification-for-ase.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Upgrade Notifications for App Service"
+author_name: "Michimune Kohno and Sanghmitra Gite"
+toc: true
+toc_sticky: true
+tags:
+    - Upgrade notifications
+    - Azure Monitor
+    - App Service Environment
+    - Logic Apps
+---
+
+We are happy to announce that the upgrade notifications for App Service Environment is available. With an upgrade notification you will be able to receive email, SMS text, or take action before a platform upgrade actually starts.
+
+The Azure App Service is a PaaS solution offering in Azure which constantly updates the platform. One of the top requests from customers is to receive a notification before an upgrade operation happens. This feature has been rolled out for App Service Environments (ASEs) across regions.
+
+> Support for public App Service scale units will be coming soon.
```

Comment:
Instead of calling it public App Service scale units, I think saying 'other App Service skus' is better as ASE is nothing but an isolated sku to customers.

Reply:
Got it.

Reply:
Updated